Cannot find module 'nan'

With a fresh git clone, after running `npm install` from the root project directory, it throws this error. [I believe its because webpack-dev-server is using an old fsevents package](https://github.com/fsevents/fsevents/issues/320). 

Environment
```
Angular CLI: 9.1.0
Node: 13.13.0
OS: Mac High Sierra 10.13.6
```

```
$ npm install

> fsevents@1.2.12 install /Users/web_connect/Development/web_connect/YoutubeDL-Material/node_modules/karma/node_modules/fsevents
> node-gyp rebuild

internal/modules/cjs/loader.js:955
  throw err;
  ^

Error: Cannot find module 'nan'
Require stack:
- /Users/web_connect/Development/web_connect/YoutubeDL-Material/node_modules/karma/node_modules/fsevents/[eval]
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:952:15)
    at Function.Module._load (internal/modules/cjs/loader.js:835:27)
    at Module.require (internal/modules/cjs/loader.js:1012:19)
    at require (internal/modules/cjs/helpers.js:72:18)
    at [eval]:1:1
    at Script.runInThisContext (vm.js:131:20)
    at Object.runInThisContext (vm.js:297:38)
    at Object.<anonymous> ([eval]-wrapper:10:26)
    at Module._compile (internal/modules/cjs/loader.js:1123:30)
    at evalScript (internal/process/execution.js:94:25) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/Users/web_connect/Development/web_connect/YoutubeDL-Material/node_modules/karma/node_modules/fsevents/[eval]'
  ]
}
```

--- a/package.json
+++ b/package.json
@@ -34,6 +34,7 @@
     "file-saver": "^2.0.2",
     "filesize": "^6.1.0",
     "fingerprintjs2": "^2.1.0",
+    "nan": "^2.14.1",
     "ng-lazyload-image": "^7.0.1",
     "ngx-file-drop": "^9.0.1",
     "ngx-videogular": "^9.0.1",

